cpp/velox/shuffle/VeloxShuffleReader.cc

```diff
@@ -314,9 +315,7 @@ VeloxShuffleReader::VeloxShuffleReader(
     std::shared_ptr<arrow::MemoryPool> pool,
     std::shared_ptr<memory::MemoryPool> veloxPool)
     : Reader(in, schema, options, pool), veloxPool_(std::move(veloxPool)) {
-  ArrowSchema cSchema;
-  GLUTEN_THROW_NOT_OK(arrow::ExportSchema(*schema, &cSchema));
-  rowType_ = asRowType(importFromArrow(cSchema));
```

Comment:
so, the leakage is here: should release cSchema ?

Reply:
Yes. It's the leakage source in this case. Gluten creates a lot of this reader instance to handle Spark's shuffle read fetches.